src/hammer-vlsi/hammer_vlsi_impl.py

```diff
@@ -221,28 +245,6 @@ def from_str(x: str) -> "PlacementConstraintType":
     def __str__(self) -> str:
         return reverse_dict(PlacementConstraintType.__mapping())[self]
 
-    class ObstructionType(Enum):
```

Comment:
You could also just set `_ignore_ = ['ObstructionType']`

Reply:
That's fair, though I think that property is only present in Python 3.7+.

Python 3.6 results in an error message:
```
Traceback (most recent call last):
  File "/tmp/test3.py", line 2, in <module>
    class A(Enum):
  File "/tmp/test3.py", line 5, in A
    _ignore_ = ["B"]
  File "/usr/lib/python3.6/enum.py", line 84, in __setitem__
    raise ValueError('_names_ are reserved for future Enum use')
ValueError: _names_ are reserved for future Enum use
```

Reply:
Though even in 3.7+ it seems like `_ignore_` isn't what we want since we can't access the ignored member:
```
>>> A
<enum 'A'>
>>> A.B
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/mnt/data/dev/edwardcwang_cpython/Lib/enum.py", line 337, in __getattr__
    raise AttributeError(name) from None
AttributeError: B
```

Reply:
I stand corrected